docs/development-guide.md

```diff
@@ -257,3 +257,37 @@ The flow for getting a fix into a release branch is:
 1. If there are any conflicts you will need to resolve them by pulling the
    branch, resolving the conflicts and force push back the branch
 1. After the CI is green, the bot will automatically merge the backport PR.
+
+### Retriggering the CI Jobs
+
+The CI Jobs gets triggered automatically on these events, such as on
+opening fresh PRs, rebase of PRs and force pushing changes to existing PRs.
+
+Right now, we also have below commands to manually retrigger the CI jobs
+1. To retrigger all the CI jobs, comment the PR with command: `/retest all`
+
+   `NOTE`:
+   * This will rerun all the jobs including the jobs which are already passed
+
+1. To retrigger a specific CI job, use command: `/retest <job-name>`
+
+   example:
+
+   ```
+   /retest ci/centos/containerized-tests
+   ```
+
+`CAUTION`: Please do not retrigger the CI jobs without an understanding of
```

Comment:
Use **Caution** in bold, not `fixed font` please

Reply:
> Just some minor changes, looks good to me otherwise. Thanks!

Sure Neils :+1: